src/Form/Checkbox.js

```diff
@@ -19,6 +19,7 @@ const CheckboxContainer = createComponent({
       margin-left: ${p => (p.horizontal ? '12px' : 0)};
       margin-top: ${p => (p.horizontal ? 0 : '4px')};
     }
+    ${p => p.style}
```

Comment:
Given how the margins are set above, this seemed like the least ornate (e.g. specifying a `margin` prop, which I considered earlier), but still effective option. Thanks for any thoughts on whether this isn't intuitive or consistent.

Reply:
for those (like me) who dont understand this dark magic:
https://github.com/styled-components/styled-components/blob/master/packages/styled-components/src/constructors/css.js
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals#Tagged_templates

Reply:
I don't think line is necessary. styled-components take a style prop by default

Reply:
Hm, I tried removing this line and the style no longer got applied.

Reply:
It looks like it's an issue with the test style in the doc. If you exchange `"margin: 0 !important"` for 0, it works without this line

Reply:
Oh thanks, I'll do that.